release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
 ensure restart before the service CA expires without requiring manual
 intervention.
 ====
+
+[[ocp-4-3-22]]
+=== RHBA-2020:2184 - {product-title} 4.3.22 Bug Fix Update
+
+Issued: 2020-05-26
+
+{product-title} release 4.3.22 is now available. The list of packages included in
+the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:2183[RHBA-2020:2183] advisory.
+The container images and bug fixes included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:2184[RHBA-2020:2184] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5099891[{product-title} 4.3.22 container image list]
+
+[[ocp-4-3-22-bug-fixes]]
+==== Bug Fixes
+
+* Previously, installations on the s390x and ppc64le architectures would not
+complete successfully because the Samples Operator was failing to report its
+version when running on those architectures.  This problem has been fixed, and
+these installations now complete successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1779934[*BZ#1779934*])
+
+* With this update, several environment variables involving TLS certificates and
+ the ETCDCTL API version are written into /root/.profile. As a result, when
```

Comment:
I'd surround `/root/.profile` with backticks.

Reply:
Oops - I intended to but forgot to format after pasting :) Good catch!